keyboards/ocean/sus/config.h

```diff
@@ -0,0 +1,51 @@
+/*Copyright 2021 Ocean
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+#define VENDOR_ID       0x9624
+#define PRODUCT_ID      0x0009
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Ocean
+#define PRODUCT         Sus
+
+/* key matrix size */
+#define MATRIX_ROWS 4
+#define MATRIX_COLS 3
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+/*                        0    1    2    3    4    5    6    7    8    9   10  11  12  13  14*/
+#define MATRIX_ROW_PINS { B5, B4, E6, D7 }
+#define MATRIX_COL_PINS { C6, D4, D0 }
+#define UNUSED_PINS
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
```

Comment:
looking at the board it looks like the diode footprints are backwards (see below). checked a [video build guide](https://www.youtube.com/watch?v=bYs-EQz-5dY) too to see if they reccommend reversing the diode but you can see they also do solder the diodes with the cathode connected to the switch and the anode to the row trace. this orientation won't work with row2col but *should* work, albeit non-conventionally, with row2col. Has this exact firmware been tested on the board?

![image](https://user-images.githubusercontent.com/1075229/127473924-fbe6433e-235d-4616-a4e0-19407a7f6207.png)

Reply:
@obosob in the picture there is an error in the polarity of the diode, and this board uses row2col. thank you very much for your attention.  ![image](https://user-images.githubusercontent.com/30220306/127482001-613b3769-bb9e-472f-bea0-353c64d46ac6.png)